file-utils/transformer/yaml2json.go

```diff
@@ -0,0 +1,67 @@
+package transformer
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/true-north-engineering/helm-file-utils/file-utils/reader"
+	"gopkg.in/yaml.v3"
+)
+
+const (
+	Yaml2JsonPrefix = "yaml2json"
+)
+
+func Yaml2JsonTransform(inputValue reader.InputValue) (reader.InputValue, error) {
+
+	result := reader.InputValue{Kind: inputValue.Kind, Value: make(map[string][]byte)}
+	jsonfile, err := YAMLToJSONFull(inputValue.Value[reader.InputKindFile])
```

Comment:
What if inputValue.Kind == reader.InputKindDir? (in case you have yaml2json+dir://path/to/dir)
You have to take this into account, iterate over input value map (files from directory) and convert each file to json.